test/evmc/README.md

```diff
@@ -1,3 +1,3 @@
 # EVMC
 
-This is an import of [EVMC](https://github.com/ethereum/evmc) version [7.0.0](https://github.com/ethereum/evmc/releases/tag/v7.0.0).
+This is an import of [EVMC](https://github.com/ethereum/evmc) version [7.1.0](https://github.com/ethereum/evmc/releases/tag/v7.1.0).
```

Comment:
No files changed in this directory.

Reply:
See the request https://github.com/ethereum/solidity/pull/8651#issuecomment-612004544